test/tools/isoltest.cpp

```diff
@@ -178,6 +194,7 @@ TestTool::Result TestTool::process()
 						AnsiColorized(cout, formatted, {BOLD, CYAN}) << "  Contract:" << endl;
 						m_test->printSource(cout, "    ", formatted);
 						m_test->printSettings(cout, "    ", formatted);
+						printOptions(cout, "    ", formatted);
```

Comment:
Can you make sure that `isoltest` won't add this to the rewritten test file when the test fails and it offers to update the expectations for you and you press `u` to accept?

Reply:
Sorry, I didn't quite understand what you meant here ... Do you mean preventing someone in the future doing this? Or do you mean making sure that it can't be reached from inside method `updateCase` ?

Reply:
Yes, I mean `updateTestCase()`.

Have you seen what happens when you run `isoltest` the expectations fail? `soltest` just fails, but `isoltest` stops and asks you what to do. One of the options is to rewrite the test case with new expectations. I just want you to make sure this part is only printed on the screen but not included in the rewritten file (because it would fail then).

Reply:
Ah, ok!
Yes, I am sure it won't update the files. I have tested locally, including updating a failed test (both with --accept-updates and manually pressing `u` when prompted).

Reply:
ok, sounds good then.